pkg/hintrunner/zero/zerohint_utils.go

```diff
@@ -45,3 +47,16 @@ func GetUint256AsFelts(vm *VM.VirtualMachine, ref hinter.ResOperander) (*fp.Elem
 
 	return low, high, nil
 }
+
```

Comment:
could you place a short comment explaining what it does / provide the formula here as well?

Reply:
sure! thanks